fetch/fetchRequest.js

```diff
@@ -33,8 +33,8 @@ function fetchRequest(method, url, args) {
       }
       return result;
     };
-    if (response.headers.get('Accept') &&
-        response.headers.get('Accept').toLowerCase().indexOf('application/json') >= 0) {
+    if (response.headers.get('Content-Type') &&
```

Comment:
Corrected, should be `Content-Type` in response.

Reply:
🤦‍♂️

Reply:
Yes, now we are able to catch that!

fyi, `response.text()` was working properly anyway for our flow, since we always do a `JSON.parse` in http https://github.com/okta/okta-auth-js/blob/master/lib/http.js#L60, this is why I haven't notice that during manual testing.

For what I have seen all our response should be application/json, so maybe should I remove the if/else?

Reply:
Seems reasonable if we're catching it downstream. Would also be curious why we have that code to handle it in the first place - my hunch is that at some point we'd occasionally return an error that was *not* a json response. This might be fixed now though.